app.js

```diff
@@ -0,0 +1,67 @@
+/* Defeine the scene for the game.  It's called example beacuse I ripped the bones
+ * of the script from a phaser 3 example.  We should rename it*/
+
+class Example extends Phaser.Scene
+{
+    /*I don't really know what this does.  I get a constructor() and that super() 
+     * allows access of the parent attributes, but I don't get this.*/
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/super
MDN is a great resource for these technical questions.